gems/smithy/spec/interfaces/client/shapes_spec.rb

```diff
@@ -0,0 +1,163 @@
+# frozen_string_literal: true
+
+describe 'Component: Shapes' do
+  shape_tests = JSON.load_file(
+    File.expand_path('../../fixtures/shapes/shape_tests.json', __dir__)
```

Comment:
Instead of having this file, which would require updates when the model updates, and also is not quite a test file but rather a reference, why not reference the model.json fixture to assert your tests? This seems very unmaintainable.

Reply:
Yeah I can take a look at that possibility